features/visitor_can_view_articles_landing_page.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Visitor can view articles on the landing page
+    As an Editor,
+    In order to catch people's interest,
+    I would like to have the articles displayed on the landing page.
+
+    Background:
+        Given the following article exists
+        | title          | body                                                             | author          | created_at                    |
+        | This is so sad | A recent report suggest that news are mostly sad. Which is sad.  | Pablo Guera     | 0018-08-16 00:00:00 UTC       |
```

Comment:
Please do not set the date in a table. If you want to be in control of the server date, please consider using a gem like [TimeCop](https://github.com/travisjeffery/timecop) or [DeLoarean](https://github.com/bebanjo/delorean)